content/en/docs/how-to/registry-authentication.md

```diff
@@ -87,6 +71,28 @@ Use `notation login` to authenticate to an OCI-compliant registry.
 notation login -u <username> <registry>
 ```
 
+## Configure environment variables to authenticate to an OCI-compliant registry
+
+{{% alert title="Warning" color="warning" %}}
+Environment variables are not secure and should not be used in a production environment for storing sensitive data such as usernames and passwords.
+{{% /alert %}}
+
+Set the `NOTATION_USERNAME` and `NOTATION_PASSWORD` environment variables to authenticate to an OCI-compliant registry.
+
+```bash
+export NOTATION_USERNAME="YOUR_REGISTRY_USERNAME"
+export NOTATION_PASSWORD="YOUR_REGISTRY_PASSWORD"
+```
+
+After the environment variables are set, you can use Notation with your registry without `notation login`.
+
+For security reasons, unset the environment variables after you log out from the registry. For example:
```

Comment:
Users may be confused with the term `log out`. They may think they need to run some command to `log out`.

Reply:
updated based on your feedback